package.json

```diff
@@ -60,6 +60,7 @@
     "babel-plugin-transform-decorators-legacy": "1.3.4",
     "babel-plugin-transform-object-rest-spread": "6.26.0",
     "babel-plugin-transform-runtime": "6.23.0",
+    "babel-preset-env": "^1.6.1",
     "babel-preset-es2015": "6.24.1",
```

Comment:
shouldn't we also be removing babel-preset-es2015

https://github.com/babel/babel/tree/master/packages/babel-preset-env

> Without any configuration options, @babel/preset-env behaves exactly the same as @babel/preset-latest (or @babel/preset-es2015, @babel/preset-es2016, and @babel/preset-es2017 together).

Reply:
@lfender6445 correct. That was an oversight on my part with these two packages. I'll update and push up. 👍